rflx/pyrflx/typevalue.py

```diff
@@ -1077,24 +1099,26 @@ def __update_simplified_mapping(self, field: Optional[Field] = None) -> None:
             self._simplified_mapping[field.name_length] = field.typeval.size
             self._simplified_mapping[field.name_first] = field.first
             self._simplified_mapping[field.name_last] = field.last
+            self._simplified_mapping[self.message_last_name] = field.last
```

Comment:
I don't understand why this is introduced...

Reply:
The `Message'Last` attribute is required to calculate the checksum. In the unverified case the end of a message is not determined by the end of the path but by the last field that is set. Since the fields are set in order `Message'Last` is also always the `'Last` of the last field that has been set.